roqoqo-aqt/src/interface.rs

```diff
@@ -34,18 +34,35 @@ const ALLOWED_OPERATIONS: &[&str; 12] = &[
 
 /// Representation for AQT backend instructions serialized to Json
 #[derive(PartialEq, Debug, Clone, serde::Serialize, serde::Deserialize)]
-#[serde(untagged)]
+#[serde(tag = "operation")]
```

Comment:
What effect does this change have?

Reply:
It ensures that when a variant is serialized, the name of the variant is used as field  "operation" in the json object.